docs/1.0-release-notes.md

```diff
@@ -0,0 +1,248 @@
+
+### Website improvements
+
+As part of the continued preparations for the 1.0 release Contour's documentation has been relocated to the https://projectcontour.io website. Specifically;
+
+* The Getting Started documentation has moved to [projectcontour.io/getting-started](https://projectcontour.io/getting-started/)
+* Guides and How-to's have moved to [projectcontour.io/guides](https://projectcontour.io/guides)
+* Versioned release documentation has moved to [projectcontour.io/docs](https://projectcontour.io/docs)
+* Project related and non-versioned documentation has moved to [projectcontour.io/resources](https://projectcontour.io/resources/)  
+
+Huge thanks to @jpeach for his work re-organising and copy editing the website content.
+
+### IngressRoute and HTTPProxy status update improvements
+
+IngressRoute and HTTPProxy status updates are now performed by the lead Contour in the deployment. The lead Contour is determined via Kubernetes' standard leader election mechanisms.
+
+If leader election is disabled, all Contours will write status back to the Kubernetes API.
+
+Fixes #1425, #1385, and many other issues with status loops over the years.
+
+### HTTPProxy and IngressRoute OpenAPIv3 schema validation
+
+Contour 1.0.0-rc.2 includes updated OpenAPIv3 schema validations. These schemas are automatically generated from the CRD documents themselves and should be more complete and consistent than the previous hand rolled versions.
+
+Fixes #513, #1414. Thanks @youngnick
+
+### TCPProxy delegation
+
+Contour 1.0.0-rc.2 now supports TCPProxy delegation. See the [relevant section](https://projectcontour.io/docs/1.0/httpproxy) in the HTTPProxy documentation.
+
+Fixes #1655.
+
+### Envoy keepalive tuning
+
+Contour 1.0.0-rc.2 addresses an issue where connections between Contour and Envoy could become stuck half-open (one side thinks the connection is open, the other side doesn't) or half-closed (one side closes the connection, the other side never gets the message).
+
+The common theme was the cluster was using an overlay network which suggested the overlay was timing out long running TCP connections. Contour 1.0.0-rc.2 configures various keep alive mechanisms to detect networking issues between Envoy and Contour. 
+
+This fix is also included in Contour 0.15.3 and later.
+
+Fixes #1744. Thanks @youngnick, @bgagnon, and @ravilr.
+
+### Contour now waits for a full cache.
+
+Contour now delays serving traffic to Envoy until each of the API informers caught up to the API server. This changes reduces the likelyhood that Envoy can connect to a Contour instance in the process of startup and thus observe an incomplete view of the cluster.
+
+Updates #1280. Thanks @jpeach and @stevesloka.
+
+### HTTPProxy CRD v1
+
+Contour 1.0.0-rc.1 promotes the HTTPProxy CRD to v1. HTTPProxy is now considered stable and our sincere hope is that with the move v1 any changes to the CRD in the future can be made in a backwards compatible manner.  
```

Comment:
Reference 1.0.0 rather than the rc?

Reply:
I think there are three mentions of httpproxy in this doc ('cos I just concatenated all the three release notes), I'll dedup and double check.